Library/Homebrew/utils/github/api.rb

```diff
@@ -208,7 +208,7 @@ def self.credentials_error_message(response_headers, needed_scopes)
         Your #{what} credentials do not have sufficient scope!
         Scopes required: #{needed_scopes}
         Scopes present:  #{credentials_scopes}
-        #{GitHub.pat_blurb}
+        #{GitHub.pat_blurb(needed_scopes.split(", "))}
```

Comment:
This should just reuse the variable before it is joined above.

Reply:
@reitermarkus you mean I should just reassign the variable **needed_scopes** on line 199 to a new variable and use it in the change I submitted is it ?

Reply:
@reitermarkus  requested changes are done